core/nwb.ophys.yaml

```diff
@@ -234,13 +234,9 @@ groups:
     shape:
     - 2
     - 3
-    doc: Location of the origin in (x, y) or (x, y, z), in the specified unit.
+    doc: Pixel location of the origin in (x, y) or (x, y, z), zero-indexed. See also reference_frame to interpret the
```

Comment:
If I understand this correctly, this changes the ``origin_coords`` from being an absolute location in ``meters`` in physical space to an (x,y,z) location in image space. Not sure if that makes sense, as it is unclear what you are translating the image relative to. Also ``origin_coords`` is a float, which would imply sub-pixel translations?

Reply:
To be clear, I think we did not fully think through `origin_coords` during its design. The image is being translated relative to whatever coordinate space is described in `reference_frame`. 

Your comment makes me think it would be better to specify the location in meters (based on some reference frame) of just the first element (0,0) or (0,0,0) of the matrix. Or maybe that is what we were thinking initially? That way it is more straightforward to describe an imaging plane that does not include the origin of the reference frame.